smalltalksrc/VMMaker/DruidJIT.class.st

```diff
@@ -4384,7 +4383,7 @@ DruidJIT >> gen_PrimitiveEqual [
 	self genPrimReturn.
 	currentBlock := self Label.
 	jump1 jmpTarget: currentBlock.
-	^ CompletePrimitive
+	^ 0
```

Comment:
Also I'd like to know the reason of this 0 here :) What is the interpreter code ignored during primitiveEqual.

Reply:
https://github.com/Alamvic/druid/issues/106